contribute.md

```diff
@@ -19,13 +19,18 @@ Pull requests should be made targeting the `master` branch. Every few weeks, the
 
 You can run the site locally on your computer while making changes.
 
+### Setup Ruby and Bundler
+
 Ensure that you have Ruby and [Bundler](http://bundler.io/) installed.
 
+    brew install ruby
+    gem install bundler
+
 ### One-time setup
 
     bundle install --path vendor/bundle
 
-_Note: If you're on Mac OS and this fails installing nokogiri, run `brew unlink xz`, install, and then `brew link xz`._
+_Note: If you're on Mac OS and this fails installing nokogiri, run `brew unlink xz`, `brew install xz`, and then `brew link xz`._
```

Comment:
This should be reverted. The install refers to running the `bundle install` command.

Reply:
reverted